sys-kernel/coreos-sources/files/4.4/0015-Overlayfs-Use-copy-up-security-hooks.patch

```diff
@@ -1,4 +1,4 @@
-From 64ef0efdd90f5aae4fae7c76783b09af53d29dfe Mon Sep 17 00:00:00 2001
+From 78e4150dc214a67ee53b9d2fc0a3d893286d21eb Mon Sep 17 00:00:00 2001
```

Comment:
0019-overlayfs-use-a-minimal-buffer-in-ovl_copy_xattr.patch is upstream in 4.4.3 (woot)

so we drop 0019 and amend this patch to accommodate the new copy_up.